checks/startupCheck.go

```diff
@@ -19,18 +19,23 @@ type LogSender interface {
 /// Register checks here
 var checks = []checkFn{
 	checkHandler,
-	sanityCheck,
```

Comment:
Did you mean to remove this, or was that a merge mistake?

Reply:
Merge mistake, re-added